test/unit/utils/Client.util.test.ts

```diff
@@ -20,12 +20,15 @@ import {
   getBulkPublishResponse,
   getClientOptions,
   createHTTPQueryParam,
-  parseEndpoint,
 } from "../../../src/utils/Client.util";
 import { Map } from "google-protobuf";
 import { PubSubBulkPublishEntry } from "../../../src/types/pubsub/PubSubBulkPublishEntry.type";
 import { PubSubBulkPublishApiResponse } from "../../../src/types/pubsub/PubSubBulkPublishApiResponse.type";
 import { CommunicationProtocolEnum, DaprClientOptions, LogLevel } from "../../../src";
+import { HttpEndpoint } from "../../../build/utils/Client.util";
```

Comment:
These all can go to Network.test.ts

Reply:
Yes, sorry, I forgot to move them when I moved the other files.